sklearnex/tests/test_memory_usage.py

```diff
@@ -94,7 +94,10 @@ def remove_duplicated_estimators(estimators_list):
     return estimators_map.values()
 
 
-BANNED_ESTIMATORS = ("TSNE",)  # too slow for using in testing on common data size
+BANNED_ESTIMATORS = (
+    "IncrementalEmpiricalCovariance",  # dataframe_f issues
+    "TSNE",
+)  # too slow for using in testing on common data size
```

Comment:
Probably, comment was moved during reformatting
```suggestion
    "TSNE",  # too slow for using in testing on common data size
)  
```